task/nexus-lifecycle-scan/0.1/nexus-lifecycle-scan-task.yml

```diff
@@ -0,0 +1,127 @@
+apiVersion: tekton.dev/v1beta1
```

Comment:
can you please rename the file to `nexus-lifecycle-scan.yaml` ?

Reply:
@vinamra28 , complete.